roles/httpd/tasks/main.yml

```diff
@@ -1,22 +1,9 @@
 ---
-- name: Install Apache httpd
-  ansible.builtin.package:
-    name:
-      - httpd
-      - mod_ssl
+- name: Pull Apache container image
+  containers.podman.podman_image:
+    name: registry.access.redhat.com/ubi9/httpd-24:latest
     state: present
 
-- name: Set httpd_can_network_connect so Apache can connect to Puma and Gunicorn
-  ansible.posix.seboolean:
-    name: httpd_can_network_connect
-    state: true
-    persistent: true
-
-- name: Disable welcome page
```

Comment:
Is the welcome page disabled in the container we would be using?

Reply:
it's not, i see it still exists, so added a task to remove it explicitly.